orange-datamodel/src/main/java/com/hartwig/hmftools/datamodel/orange/OrangeRecord.java

```diff
@@ -31,9 +31,9 @@ public interface OrangeRecord {
     String sampleId();
 
     @NotNull
-    LocalDate experimentDate();
+    LocalDate samplingDate();
 
-    @Nullable
+    @NotNull
     ExperimentType experimentType();
```

Comment:
@kduyvesteyn is experiment still a good term here? Perhaps SequencingType is more descriptive?

Reply:
I discussed with Ewart. The term "experiment" is used in our lab extensively but is an internal term. E.g. when the lab isolates DNA from a biopsy they call this a "DNA isolation experiment". 

I have asked whether SequencingType would be a better name but both targeted and WGS are sequenced identically (just with different libraries), so then SequencingType isn't a better name either. 

I can't think of a better name than ExperimentType at the moment, it relates quite well (but not perfectly) with the naming conventions in the lab

Reply:
Ok I'm satisfied!